sections/home.mdx

```diff
@@ -12,12 +12,13 @@ order: 0
 
 <Content spaceTop spaceBottom>
 
-# Animated Drawings
+## Animated Drawings
 
 This repo contains an implementation of the algorithm described in the paper, `A Method for Automatically Animating Children's Drawings of the Human Figure' (currenly under peer review). 
+
 In addition, this repo aims to be a useful creative tool in its own right, allowing you to flexibly create animations starring your own drawn characters.
 
-If you do create something fun with this, let us know! Use hashtag **#FAIRAnimatedDrawings**
+If you do create something fun with this, let us know! Use hashtag [**#FAIRAnimatedDrawings**](https://twitter.com/search?q=%23ChatGPT)
```

Comment:
Good idea to hyperlink. I'm modifying it to be https://twitter.com/search?q=%23FAIRAnimatedDrawings ?

Reply:
Haha oops wrong link! Thanks for catching it.